src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -28,6 +33,9 @@
 
 public final class DatasetJsonConversion {
 
+    private static GoogleRegion DEFAULT_GOOGLE_REGION = GoogleRegion.US_CENTRAL1;
+    private static CloudPlatform DEFAULT_CLOUD_PLATFORM = CloudPlatform.GCP;
```

Comment:
That's a great idea, but the `DatasetJsonConversion` is a static class and not an autowired component, which means it doesn't have access to the `application.properties` or any of the `Configuration` classes that get populated by it. I think this would be a good change, but is a little out of scope for this PR given all of the touch points. We can make a ticket and add it to this epic. Thoughts?

Reply:
Eh, I don't know that's it worth it.

Reply:
I think the only reason we'd want it in application.properties would be if we wanted to set the default per environment or something like that. I don't see a use case for that any time soon, and even if it became one, it'd be simple to make that change in the future.